### PR TITLE
Users

### PR DESCRIPTION
**Pull request by danieljtate**
connects to #24

### CODE REVIEW

File: app/components/Users.js [hunk 1]
```diff
@@ -1,13 +1,623 @@
...
+    }
+
+    this.setState(function(previousState){
+      return {
+        teams: newArr
+      }
+    })
+  },
+
+  checkAll: function(e){
```
Comment (AlexJYoungman) [inline app/components/Users.js]: All function methods should have a space before the parameters. See Below.
`function (e) {`

File: app/components/Users.js [hunk 2]
```diff
@@ -1,13 +1,623 @@
...
+
+for(var i in this.state.filterTeams) {
+    if(newProjects.indexOf(this.state.filterTeams[i].Project) == -1) {
+      newProjects.push(this.state.filterTeams[i].Project);
+    }
   }
-});
 
-module.exports = Users;
+  var filterTeamsTableProject = newProjects.map(function(obj, idx) {
```
Comment (AlexJYoungman) [inline app/components/Users.js]: Variable name makes function unclear

File: app/components/Users.js [hunk 3]
```diff
@@ -1,13 +1,623 @@
...
+                        </td>
+                      </tr>
+                      <tr>
+                        <td>Project(s)</td>
+                        <td><select id="addUserFormField">
+                            <option selected disabled>Please Choose...</option>
+                            <option value="SmartCare">Smart Care</option>
+                            <option value="BlockChain">Block Chain</option>
+                            <option value="Ab-ccenture">Ab-ccenture</option>
+                            <option value="Robotic Mortgage Advisor">Robotic Mortgage Advisor</option>
```
Comment (AlexJYoungman) [inline app/components/Users.js]: Should these values not be taken from the json data, instead of being static?

File: app/components/Users.js [hunk 4]
```diff
@@ -1,13 +1,623 @@
...
+  var newTeams = [];
+
+for(var i in this.state.filterTeams) {
+    if(newTeams.indexOf(this.state.filterTeams[i].Team) == -1) {
+      newTeams.push(this.state.filterTeams[i].Team);
+    }
+  }
+
+
+  var filterTeamsTableTeams = newTeams.map(function(obj, idx) {
```
Comment (AlexJYoungman) [inline app/components/Users.js]: Unclear variable name

File: app/components/Users.js [hunk 5]
```diff
@@ -1,13 +1,623 @@
...
+  checkAll: function(e){
+    //when clicking the checkbox in the table header this will check all boxes in the table
+    if(document.getElementById("allUsersCheckbox").checked)
+    {
+      var elems = document.getElementsByClassName("userCheckbox");
+      for(var i = 0; i < elems.length; i++) {
+        elems[i].checked = true;
+      }
+    }
+    else {
```
Comment (AlexJYoungman) [inline app/components/Users.js]: Try to be consistent with your bracket placement
```
if () {
} else {
}
```
or
```
if ()
{
}
else 
{
}
```

File: app/components/Users.js [hunk 6]
```diff
@@ -1,13 +1,623 @@
...
+        elems[i].checked = false;
+      }
+    }
+    this.checkme();
+  },
+
+  componentDidMount: function() {
+    /*Ensures that the Assign buttons are disabled by default on page load */
+    document.getElementById("assignTeamButton").disabled=true;
+    document.getElementById("assignProjectButton").disabled=true;
```
Comment (AlexJYoungman) [inline app/components/Users.js]: Try and use spaces around assignments.
`variable = value;` not `variable=value;`

File: app/components/Dashboard.js [hunk 7]
```diff
@@ -1,7 +1,5 @@
 var React = require('react');
 //var ReactDOM = require('react-dom');
```
Comment (TheGarrett) [inline app/components/Dashboard.js]: remove commented out code if not needed

File: app/components/Dashboard.js [hunk 8]
```diff
@@ -1,7 +1,5 @@
...
-var GChart = require('google-chart-react');
-//var GChart = require('react-google-charts');
 var d3 = require('d3');
```
Comment (TheGarrett) [inline app/components/Dashboard.js]: Indentation around lines 267.

File: app/styles/css/users.css [hunk 9]
```diff
@@ -362,20 +374,21 @@ padding-bottom:-5px;
   margin-top: -25px;
 }
 
-#thing {
-display:none;
+#usersCheckbox {
+display:hidden;
```
Comment (TheGarrett) [inline app/styles/css/users.css]: indentation and same for below

**Pull request merged**